integration-tests/rpc-get-protocol-constants.spec.ts

```diff
@@ -508,43 +510,19 @@ CONFIGS().forEach(({ lib, protocol, rpc }) => {
                     numerator: 1,
                 },
                 minimal_block_delay: new BigNumber(15),
-                delay_increment_per_round: new BigNumber(7),
-                cache_layout_size: 3,
-                cache_sampler_state_cycles: 8,
-                cache_script_size: 100000000,
-                cache_stake_distribution_cycles: 8,
-                cycles_per_voting_period: 1,
-                sc_rollup_challenge_window_in_blocks: 20160,
-                sc_rollup_enable: false,
-                sc_rollup_max_available_messages: 1000000,
-                sc_rollup_origination_size: 6314,
-                tx_rollup_commitment_bond: new BigNumber(10000000000),
-                tx_rollup_cost_per_byte_ema_factor: 120,
-                tx_rollup_enable: true,
-                tx_rollup_finality_period: 40000,
-                tx_rollup_hard_size_limit_per_inbox: 500000,
-                tx_rollup_hard_size_limit_per_message: 5000,
-                tx_rollup_max_commitments_count: 80100,
-                tx_rollup_max_inboxes_count: 40100,
-                tx_rollup_max_messages_per_inbox: 1010,
-                tx_rollup_max_ticket_payload_size: 2048,
-                tx_rollup_max_withdrawals_per_batch: 15,
-                tx_rollup_origination_size: 4000,
-                tx_rollup_rejection_max_proof_size: 30000,
-                tx_rollup_sunset_level: 3473409,
-                tx_rollup_withdraw_period: 40000,
+                delay_increment_per_round: new BigNumber(5)
             });
 
             done();
         })
 
-        kathmandunet(`successfully fetches all constants for kathmandunet using ${rpc}`, async (done) => {
```

Comment:
test for kathmandunet should not be removed

Reply:
i had to merge with master to get the kathmandunet stuff back into the branch